app/models/node.rb

```diff
@@ -20,6 +20,8 @@ class Node < ApplicationRecord
   enumerize :state, in: STATES, scope: true
   content_attribute :content_body
   content_attribute :name
+  # options is not currently versioned but could be in the future
+  store_attribute :content, :options
```

Comment:
If we were to version this stuff, we wouldn't be using diffs.... so might be cleaner to keep it separate even if we later extend Revision to include values for options etc...